### PR TITLE
fix #76875 moviesflix.one

### LINKED ISSUE #76875: moviesflix.one
### Issue URL (Ads)
 [https://moviesflix.one](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fmoviesflix.one)
### Comment
Username: @iamthelara
> 
> --
> The information below is automatically generated. Do not delete it.
> 
> Custom userscripts:
> https://userscripts.adtidy.org/release/adguard-extra/1.0/adguard-extra.meta.js
> 

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | Android 10
AdGuard product: | AdGuard for Android v3.6.1 (11)
Browser: | Chrome
AdGuard mode: | VPN
Filtering quality: | High-quality
HTTPS filtering: | enabled
DNS filtering: | server: `Ad Guard DNS(DNS over QUIC)`<br>filters: `Ad Guard DNS filter`
Filters: | <b>Ad Blocking:</b><br/> AdGuard Mobile Ads, <br/>AdGuard Base, <br/>Peter Lowe's Blocklist<br/><br/><b>Privacy:</b><br/> EasyPrivacy, <br/>Fanboy's Enhanced Tracking List, <br/>AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> Fanboy's Annoyances, <br/>Adblock Warning Removal List, <br/>Web Annoyances Ultralist, <br/>I don't care about cookies, <br/>AdGuard Annoyances<br/><br/><b>Security:</b><br/> Online Malicious URL Blocklist<br/><br/><b>Other:</b><br/> AdGuard DNS
Other extensions used: | Nope

### PR DESCRIPTION
**Pull request by wout-codes**
#76875

`||blogads*.blogspot.com^$third-party` with `*` because there are many sites which makes the number (001, etc) a variable. For example, `blogads002.blogspot.com`, `blogads003.blogspot.com`, in that site.

Also added, this domain to the `|https://$script,subdocument,third-party,xmlhttprequest,domain=` to prevent further ads from loading. No breakages too.

### CODE REVIEW

File: EnglishFilter/sections/specific.txt [hunk 1]
```diff
@@ -4904,7 +4904,7 @@ hdzog.com##.main-content > .block-thumbs:not([class*=" "])
 4tube.com###ad_player
 ||shaggyimg.pro/crackle/carouseal.js
 gamefaqs.gamespot.com##.message_mpu
-|https://$script,subdocument,third-party,xmlhttprequest,domain=upload4earn.org
+|https://$script,subdocument,third-party,xmlhttprequest,domain=upload4earn.org|moviesflix.one
```
Comment (Alex-302) [inline EnglishFilter/sections/specific.txt:4907]: mirror: themoviesflix.co
better add to regex rule 
https://github.com/AdguardTeam/AdguardFilters/commit/730a983e7b86598456b8fdbe28ee3817d50920a2#diff-8e737f6429801c4aac12f0a8ddb7b4ab3173f39642bfe56004e22dede37edd6dR217

**Review by Alex-302 — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:4926]: @Alex-302 It doesn't do anything except blocking one ad-server. It doesn't block the others.

Comment (wout-codes) [reply EnglishFilter/sections/specific.txt:4926]: Okay.

**Review by Alex-302 — APPROVED** [review]

File: EnglishFilter/sections/specific.txt (hunk 1 above)
Comment (Alex-302) [reply EnglishFilter/sections/specific.txt]: Missed the mirror - themoviesflix.co 
Added in my commit

**Pull request merged**